src/adhocracy_core/adhocracy_core/content/__init__.py

```diff
@@ -83,6 +83,39 @@ def resources_meta_addable(self):
             resources_addables[iresource] = all_addables
         return resources_addables
 
+    @property
+    def permissions(self) -> [str]:
+        """Set of all permissions defined in the system."""
+        perms = {'hide_resource', 'edit_group', 'do_transition'}  # built-in permissions
```

Comment:
if you extract this to a property its easier to extand and you don't neet the comment to explain the meaning.